daemon/events/filter.go

```diff
@@ -29,6 +29,24 @@ func (ef *Filter) Include(ev events.Message) bool {
 		ef.matchLabels(ev.Actor.Attributes)
 }
 
+func (ef *Filter) matchEvent(ev events.Message) bool {
+	// #25798 if an event filter contains either health_status, exec_create or exec_start without a colon
+	// Let's to a FuzzyMatch instead of an ExactMatch.
+	if ef.filterContains("event", map[string]struct{}{"health_status": {}, "exec_create": {}, "exec_start": {}}) {
+		return ef.filter.FuzzyMatch("event", ev.Action)
```

Comment:
We should really rename `FuzzyMatch` to `PrefixMatch` as it's not doing anything "fuzzy", only optimizing a prefix match